resources/hab_build.rb

```diff
@@ -31,38 +37,25 @@
 action :publish do
   execute 'upload-pkg' do
     command lazy { "#{hab_binary} pkg upload --url #{url} #{hab_studio_path}/src/results/#{artifact}" }
-    env('HOME' => home_dir,
-        'HAB_AUTH_TOKEN' => auth_token,
-        'HAB_NONINTERACTIVE' => 'true')
+    env(
+      'HOME' => home_dir,
+      'HAB_AUTH_TOKEN' => auth_token,
+      'HAB_NONINTERACTIVE' => 'true'
+    )
     live_stream new_resource.live_stream
     sensitive true
   end
 
-  load_delivery_chef_config
-  chef_data_bag new_resource.name
-
-  # We need to be clear here to ensure that the magic done by cheffish
-  # doesn't cause any unexpected data bag item naming or contents.
-  # Thus, we have a unique project name for the resource, which does
-  # not get used because we're setting the `id` and the `data_bag`
-  # properties. We need to use `lazy {}` on the `id` and `raw_data`
-  # because those contain calculated values from a previously
-  # converged resource, `hab_build`, above.
-  #
-  chef_data_bag_item "store-#{new_resource.name}-artifact-data" do
-    id lazy { build_version }
-    data_bag new_resource.name
-    raw_data lazy {
-      { 'id' => build_version,
-        'version' => build_version,
+  ruby_block "create Habitat project release: #{new_resource.name} #{build_version}" do
+    block do
+      # This helper is part of the Delivery Sugar DSL...it's also an alias
+      # for `define_project_application`.
+      create_workflow_application_release(
+        new_resource.name,
+        build_version,
```

Comment:
[delivery-sugar is now updated](https://github.com/chef-cookbooks/delivery-sugar/commit/55fbde9608b66341e6b102a8e1d6bceece7f0d55) to allow build versions with forward slashes so we can use the Habitat-native version like `MAJOR.MINOR.PATCH/YYYYMMDDHHMMSS` when creatting an application release.